### PR TITLE
Use backtrace debugging with UNO R4 WiFi [HC-1341]

### PR DESCRIPTION
**Pull request by seaxwi**
Adds an article documenting the [CmBacktrace](https://github.com/armink/CmBacktrace) serial output for runtime errors on UNO R4 WiFi.

### CODE REVIEW

File: content/Hardware Support/UNO/Use-stack-backtrace-to-debug-a-sketch-for-UNO-R4-WiFi.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+1. Look at the last line in the serial monitor output, which starts with `Show more call stack info by run:`.
+2. Copy the `addr2line` command. Make sure to include the full command, for example:
+
+   `addr2line -e "/private/var/folders/58/gntldnl9249ck9fjblgv9jdw0000gp/T/arduino/sketches/EE268ECDDB1E91C34016236A92C09AA0/ErrorInducer.ino".elf -a -f 00004188 00008718 0000426e 00009bd4 00009c12 00004296 0000753a 00007530`
+
+3. Open Terminal (macOS, Linux) or [Windows Powershell](https://learn.microsoft.com/en-us/powershell/scripting/windows-powershell/starting-windows-powershell?view=powershell-7.3).
+   * **Windows only:** Type `Ubuntu` into Windows Powershell and press <kbd>Enter</kbd>.
+4. Paste the addr2line command and press <kbd>Enter</kbd>.
+5. **Windows only:** Paste the command in a text editor. Replace "C:" with "/mnt/c" and replace **all** backslashes (\\) with forward slashes (/).
```
Comment (Renat0Ribeir0) [inline content/Hardware Support/UNO/Use-stack-backtrace-to-debug-a-sketch-for-UNO-R4-WiFi.md:31]:
```suggestion
3. **Windows only:** Paste the command in a text editor. Replace "C:" with "/mnt/c" and replace **all** backslashes (\\) with forward slashes (/).
4. Open Terminal (macOS, Linux) or [Windows Powershell](https://learn.microsoft.com/en-us/powershell/scripting/windows-powershell/starting-windows-powershell?view=powershell-7.3).
   * **Windows only:** Type `Ubuntu` into Windows Powershell and press <kbd>Enter</kbd>.
5. Paste the `addr2line` command (or the `addr2line` command altered in step 3, if using **Windows**) and press <kbd>Enter</kbd>.
```
Is altering the addr2line command in a text editor necessary before pasting it into the Powershell? If so, I would consider restructuring the list in this order.

File: content/Hardware Support/UNO/Use-stack-backtrace-to-debug-a-sketch-for-UNO-R4-WiFi.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+* A serial connection must be initiated before the error occurs, by calling [Serial.begin()](https://www.arduino.cc/reference/en/language/functions/communication/serial/begin/) in your sketch.
+* The Serial Monitor must be open.
+* `addr2line` must be installed on your system:
+  * **Windows:** `addr2line` is not available as a native Windows application, but can be run with [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install). The Ubuntu distribution of Linux is installed by default and should come with `addr2line`. Otherwise, install it with `apt-get install binutils`.
+  * **macOS:** `addr2line` can be installed with [Homebrew](https://brew.sh/) by running `brew install binutils` in Terminal.
+  * **Linux:** `addr2line` may already installed on your system. Otherwise, run `apt-get install binutils` in Terminal (Ubuntu, Debian), or see [command-not-found.com/addr2line](https://command-not-found.com/addr2line) for other distributions.
+
+## Instructions
+
+Follow these steps to run the addr2line command:
```
Comment (Renat0Ribeir0) [inline content/Hardware Support/UNO/Use-stack-backtrace-to-debug-a-sketch-for-UNO-R4-WiFi.md:21]:
```suggestion
Follow these steps to run the `addr2line` command:
```
Formatting

**Review by Renat0Ribeir0 — CHANGES_REQUESTED** [review]

**Review by Renat0Ribeir0 — APPROVED** [review]
LGTM 👍

**Comment by seaxwi** [comment]
@Renat0Ribeir0 Made some small additional changes, could you have a look?

**Review by Renat0Ribeir0 — APPROVED** [review]

**Pull request merged**